installation/sql/mysql/extensions.sql

```diff
@@ -882,3 +882,93 @@ CREATE TABLE IF NOT EXISTS `#__action_logs_users` (
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `#__guidedtours`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtours` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `asset_id` int DEFAULT 0,
+  `title` varchar(255) NOT NULL,
+  `description` text NOT NULL,
+  `ordering` int NOT NULL DEFAULT 0,
+  `extensions` text NOT NULL,
+  `url` varchar(255) NOT NULL,
+  `overlay` tinyint NOT NULL  DEFAULT 0,
+  `created` datetime NOT NULL,
+  `created_by` int NOT NULL DEFAULT 0,
+  `modified` datetime NOT NULL,
+  `modified_by` int NOT NULL DEFAULT 0,
+  `checked_out_time` datetime NOT NULL,
+  `checked_out` int NOT NULL DEFAULT 0,
+  `published` tinyint NOT NULL DEFAULT 0,
+  `state` tinyint NOT NULL DEFAULT '1',
+  PRIMARY KEY (`id`),
+  KEY `idx_asset_id` (`asset_id`),
+  KEY `idx_title` (`title`(191)),
+  KEY `idx_created` (`created`),
+  KEY `idx_created_by` (`created_by`),
+  KEY `idx_modified` (`modified`),
+  KEY `idx_modified_by` (`modified_by`),
+  KEY `idx_checked_out` (`checked_out`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
+
+--
+-- Dumping data for table `#__guidedtours`
+--
+
+INSERT INTO `#__guidedtours` (`id`, `asset_id`, `title`,`description`, `ordering`, `extensions`, `url`,`overlay`, `created`, `created_by`, `modified`, `modified_by`, `checked_out_time`, `checked_out`, `published`, `state`) VALUES
+(1, 91, 'How to create a Guided Tour in Joomla Backend?','<p>This Tour will show you that how you can create Guided Tour in Joomla Backend!</p>', 0, '[\"com_guidedtours\"]', 'http://localhost/soc21_guided-tour/administrator/index.php?option=com_guidedtours',1, '2021-09-17 18:33:56', 430, '2021-09-17 18:33:56', 430, '2021-09-17 18:46:24', 430, 1, 1);
+
+
+-- --------------------------------------------------------
+  
+--
+-- Table structure for table `#__guidedtour_steps`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtour_steps` (
+  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `tour_id` int NOT NULL DEFAULT 0,
+  `title` varchar(255) NOT NULL,
+  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
+  `published` tinyint NOT NULL DEFAULT 0,
+  `description` text NOT NULL,
+  `ordering` int NOT NULL DEFAULT 0,
+  `step-no` int NOT NULL DEFAULT 0,
+  `position` varchar(255) NOT NULL,
+  `target` varchar(255) NOT NULL,
+  `created` datetime NOT NULL,
+  `created_by` int unsigned NOT NULL DEFAULT 0,
+  `modified` datetime NOT NULL,
+  `modified_by` int unsigned NOT NULL DEFAULT 0,
+  `state` tinyint NOT NULL DEFAULT '1',
+  KEY `idx_tour` (`tour_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
+
+--
+-- Dumping data for table `#__guidedtour_steps`
+--
+
+INSERT INTO `#__guidedtour_steps` (`id`, `tour_id`, `title`, `published`, `description`, `ordering`, `step-no`, `position`, `target`, `created`, `created_by`, `modified`, `modified_by`, `state`) VALUES
```

Comment:
@jatinS-dev Similar to the #__guidedtour table notes, the steps all need to be cleaned out for the following fields:

`created`, `created_by`, `modified`, `modified_by`